public/flag.js

```diff
@@ -0,0 +1,5 @@
+$(window).on( "load", function() {
```

Comment:
You can add this to locations-list-map, instead of creating a new JS file. There's already logic to get the current country code in that file.